src/bin/tikv-server.rs

```diff
@@ -198,6 +198,14 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     };
     opts.set_min_write_buffer_number_to_merge(min_write_buffer_number_to_merge as i32);
 
+    let max_background_threads = get_integer_value("",
```

Comment:
DBOptions* DBOptions::IncreaseParallelism(int total_threads) {
  max_background_compactions = total_threads - 1;
  max_background_flushes = 1;
  env->SetBackgroundThreads(total_threads, Env::LOW);
  env->SetBackgroundThreads(1, Env::HIGH);
  return this;
}
This is rocksdb's code, as we can see, IncreaseParallelism affect max_backgroud_compactions and max_backgroud_flushes, and max_backgroud_compactions is configurable in our system, and max_backgroud_flushes by default is 1, actually we don't need this option. You are welcome to make max_backgroud_flushed configurable. Thanks